app/models/comment.rb

```diff
@@ -1,2 +1,12 @@
 class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: :author_id
+  belongs_to :post
+
```

Comment:
- [ ] Kindly note that a comment belongs to the author and post. So please change line `2` to `belongs_to :author, class_name: 'User'`